components/filter/filter.js

```diff
@@ -90,8 +99,12 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		const header = this._buildHeader(singleDimension);
 		const dimensions = this._buildDimensions(singleDimension);
 
-		const buttonText = singleDimension ? this._dimensions[0].text : this.localize('components.filter.filters');
-		const description = singleDimension ? this.localize('components.filter.singleDimensionDescription', { filterName: this._dimensions[0].text }) : this.localize('components.filter.filters');
+		const filterCount = this._formatFilterCount(this._totalAppliedCount, this._totalMaxCount);
+		let buttonText = singleDimension ? this._dimensions[0].text : this.localize('components.filter.filters');
```

Comment:
Nit: seems like we can avoid some string concatenation.  I think it could be avoided in a single statement as a `const`, but also in two statements if readability is desired.  I think the two statement version would be something like:
```javascript
let buttonText = singleDimension ? this._dimensions[0].text : this.localize('components.filter.filters');
if (filterCount) buttonText = `${buttonText} (${filterCount})`;
```